pyarr/models/common.py

```diff
@@ -1,25 +1,23 @@
+from dataclasses import dataclass
 from enum import Enum
 
-import enum_tools.documentation
 
-enum_tools.documentation.INTERACTIVE = True
-
-
-@enum_tools.documentation.document_enum
-class PyarrSortDirection(str, Enum):
+@dataclass(order=True)
```

Comment:
On second though, `order` might actually not be needed here, since the key values are now just strings.